src/main/java/Duke.java

```diff
@@ -1,10 +1,91 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static void printHorizontalLine() {
+        String horizontal_line = "____________________________________________________________";
```

Comment:
```suggestion
        String horizontalLine = "____________________________________________________________";
```
Perhaps use horizontalLine instead of horizontal_line so that it will be in camelCase?

Reply:
Thanks!